setup.py

```diff
@@ -27,11 +27,6 @@
     "semantic_version==2.6"
 ]
 
-extra_requirements = {
-    'pytorch':  ["torch", "torchvision"],
-    'tf':  ["tensorflow>=1.12"],
-}
-

```

Comment:
These were never used in our installation guide, and will now no longer work, as pytorch has moved to hosting their own pip repository.